dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageProblem.java

```diff
@@ -150,9 +150,21 @@ public String formatSymbolProblem() {
   public static String formatLinkageProblems(Set<LinkageProblem> linkageProblems) {
     StringBuilder output = new StringBuilder();
 
+    // Not to group AbstractMethodProblems by symbols because they do not fit in the
+    // "... referenced by ..." format.
+    ImmutableSet.Builder<AbstractMethodProblem> abstractMethodProblems = ImmutableSet.builder();
+    ImmutableSet.Builder<LinkageProblem> problemsToGroupBySymbols = ImmutableSet.builder();
+    for (LinkageProblem linkageProblem : linkageProblems) {
+      if (linkageProblem instanceof AbstractMethodProblem) {
+        abstractMethodProblems.add((AbstractMethodProblem) linkageProblem);
+      } else {
+        problemsToGroupBySymbols.add(linkageProblem);
+      }
+    }
```

Comment:
When printing linkage problems, we group them by symbols with "... referenced by ..." format. With this PR, AbstractMethodProblem are printed without the grouping.